source/_data/glossary.yml

```diff
@@ -9,12 +9,16 @@
 #
 
 - term: Action
-  definition: >-
-    An action is an command that can be fired. For example, turning on a light.
-    Actions used in many places, most notably in automations and scripts.
+  definition: |-
+    An action is a part of a script or automation that defines what is going to happen once a trigger is activated. In scripts, an action is called *sequence*.
```

Comment:
Not sure if it matters, but they are not limited to scripts or automations.

Reply:
Yes... I felt the need to provide more detail on "used in many places", as this felt a bit vague. But I did not know what to write, so I chickened out. Sent a new proposal now.